handsontable/src/helpers/dom/element.js

```diff
@@ -1194,3 +1194,15 @@ export function runWithSelectedContendEditableElement(element, callback, invisib
 
   removeContentEditableFromElementAndDeselect(element, invisibleSelection);
 }
+
+/**
+ * Check if the element is HTMLElement.
+ *
+ * @param {HTMLElement|number} element Element to check.
+ * @returns {boolean} `true` if the element is HTMLElement.
+ */
+export function isHTMLElement(element) {
+  const OwnElement = element?.ownerDocument?.defaultView?.Element;
```

Comment:
```suggestion
  const OwnElement = element?.ownerDocument.defaultView.Element;
```
I guess once the element exists, the `ownerDocument` is filled with all necessary properties.

Reply:
The tests started failing, so we can remove question mark only before `Element`